ppb/engine.py

```diff
@@ -73,12 +89,20 @@ def run(self):
 
     @property
     def current_scene(self):
+        """
+        Get current scene
+        """
         try:
             return self.scenes[-1]
         except IndexError:
             return None
 
     def activate(self, next_scene: dict):
+        """
+        Activates a specific scene
+
+        next_scene -- scene to be activated
```

Comment:
I'd like the expected structure of next_scene to be documented here:

* scene: a Type
* args: a list or tuple
* kwargs: a dictionary

This is important for end users, because this is one of the primary interfaces for scenes to replace themselves.